Added property for autocorrection

Tantalum73 opened the pull request:
By adressing issue #67, I added a property to set the UITextAutocorrectionType on the underlaying UITextField / UITextView.

Is is still turned off by default but the user now has the choice to turn it on.

Tantalum73 committed:
Added property for autocorrection

By adressing issue #67, I added a property to set the UITextAutocorrectionType on the underlaying UITextField / UITextView.

Is is still turned off by default but the user now has the choice to turn it on.

victorBaro commented on AnimatedTextInput/Classes/AnimatedTextInput.swift:
```diff
@@ -23,7 +23,13 @@ open class AnimatedTextInput: UIControl {
+    
+    open var autocorrection: UITextAutocorrectionType = .no {
+        didSet {
+            configureType()
```
Is this needed? `configureType()` removes the whole view from its superview and generates a new input. I would simply do `textInput.autocorrection = autocorrection`.

Tantalum73 replied:
I agree, you are right.
I made the proposed change and will do the commit soon.

victorBaro reviewed (changes requested):
Thank you for submitting this PR, I really like it, nice and simple.
Please check my suggested change.

Tantalum73 committed:
Only autocorrection value will be updated when set

Not the entire text input is re-created but only its autocorrection property is updated when a change to the value is made.

Tantalum73 commented:
Would you please review the changes that I've made after your correction-suggestion?
